src/components/ModuleFields/Configurator/File.vue

```diff
@@ -71,6 +223,10 @@ export default {
     enableFileNameHiding () {
```

Comment:
You can rename this method instead of creating a duplicate one with a different name (`enablePreviewStyling`)